docs/reference/setup/install/create-certs.yml

```
@@ -3,7 +3,7 @@ version: "2.2"
 services:
   create_certs:
     container_name: create_certs
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.15.1
+    image: docker.elastic.co/elasticsearch/elasticsearch:{VERSION}
```

Styling nit. It looks like it still works with caps though.
```suggestion
    image: docker.elastic.co/elasticsearch/elasticsearch:{version}
```